pkg/ocm/client.go

```diff
@@ -32,7 +32,7 @@ var log = logf.Log.WithName("ocm-client")
 
 var (
 	// ErrClusterIdNotFound is an error describing the cluster ID can not be found
-	ErrClusterIdNotFound = fmt.Errorf("cluster ID can't be found")
+	ErrClusterIdNotFound = fmt.Errorf("OCM did not return a valid cluster ID: pull-secret may be invalid")
```

Comment:
Thank you for The PR. Thinking if we could add Cluster's owner-banned scenario in the log info for more clarity. ref: https://access.redhat.com/articles/1340183
```suggestion
	ErrClusterIdNotFound = fmt.Errorf("OCM did not return a valid cluster ID: pull-secret may be invalid OR cluster's owner is disabled/banned in OCM")
```